CMakeLists.txt

```
@@ -1,16 +1,18 @@
 cmake_minimum_required(VERSION 3.19)
 project(my_application LANGUAGES CXX)
 
```

For use later, to show when the optional package was found:

```suggestion
include(FeatureSummary)
```